lighthouse-core/audits/themed-omnibox.js

```diff
@@ -43,13 +43,13 @@ class ThemedOmnibox extends MultiCheckAudit {
   }
 
   /**
-   * @param {LH.Artifacts['ThemeColor']} themeColorMeta
+   * @param {LH.Artifacts['MetaElements'][0]|undefined} themeColorMeta
```

Comment:
should we make an alias of `LH.Artifacts.MetaElement` since we use this so often?

Reply:
this is the only usage of it, but sure :)

Reply:
> this is the only usage of it, but sure

ha, whoops